test/browser_tests/spec_suites/content/blog.js

```diff
@@ -134,4 +134,35 @@ describe( 'The Blog Page', function() {
   it( 'should include a page input with value set to 1', function() {
     expect( page.paginationPageInput.getAttribute( 'value' ) === 1 );
   } );
+
+  it( 'should set an initial height on filter expandable that does not cut off submit button when page loads with 5 filter options selected', function() {
+    // open filter and select 5 options
+    page.searchFilterShowBtn.click();
+    browser.sleep( 1000 );
```

Comment:
I think we were moving away from using `browser.sleep`, if possible.  Might be nice to try 
`browser.isVisible` like https://github.com/cfpb/cfgov-refresh/blob/e0e0f874adb344f5afdd47a6404a7acdb5004478/test/browser_tests/spec_suites/organisms/mega-menu.js#L80.


https://github.cfpb.gov/CFGOV/platform/issues/40.